pkg/cloud/services/eks/nodegroup.go

```diff
@@ -195,6 +190,13 @@ func (m *NodegroupService) CreateLaunchTemplateData(ec2Svc *ec2svc.Service) (*ec
 		UserData:     pointer.StringPtr(base64.StdEncoding.EncodeToString(userData)),
 	}
 
+	sgIDs, err := m.scope.CoreSecurityGroups(scope)
```

Comment:
Using `CoreSecurityGroups` is good and this is the right pot but we can then remove the code under `func remoteAccess` (above the code you modified there) that also adds these security groups.